File: 240110/두 정수에 대한 연산값/operational-values-for-two-integers.py
a,b = map(int,input().split())

def min_max(x,y):
    if x>y :
        x = x+25
        y = y*2
    else:
        x = x * 2
        y = y + 25
    return x,y
print(min_max(a,b))

File: 240110/두 정수에 대한 연산값/operational-values-for-two-integers.java
import java.util.Scanner;
public class Main {
    public static void min_max (int x, int y){
        if (x>y){
            x = x+25;
            y = y*2;
        }
        else{
            x = x*2;
            y = y+25;
        }
        System.out.print(x+" "+y);
    }
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        min_max(a,b);

    }
}